src/registrar/templates/base.html

```diff
@@ -129,7 +129,7 @@
         {% block logo %}
         <div class="usa-logo" id="extended-logo">
           <em class="usa-logo__text">
-            <a href="/" title="Home" aria-label="Home">
+            <a href="{% url 'home' %}" title="Home" aria-label="Home">
```

Comment:
**comment**: Thank you for all of these changes to using named URLs in the templates!